NUAttribute.js

```diff
@@ -174,4 +189,21 @@ export default class NUAttribute extends NUObject {
         }
         return null;
     }
+
+    validateNumberValue(attrValue, attrObj, validateSubType) {
+        const expectedType = validateSubType ? attrObj.subType : attrObj.attributeType;
+        if (isNaN(attrValue) || (expectedType === NUAttribute.ATTR_TYPE_INTEGER && !Number.isInteger(attrValue))) {
```

Comment:
Number.isInteger (1.0) is acceptable as a number however server cannot accept this as an integer

Reply:
@natabal we use Number(value) in https://github.com/nuagenetworks/js-bambou/blob/master/NUAbstractModel.js#L31
So for such cases, though `1.0` is entered in form, the value sent to server will be `1`.
Do we still need to enhance validation?

Reply:
![Screen Shot 2020-03-31 at 1 44 26 PM](https://user-images.githubusercontent.com/4873481/78073450-c4d61400-7355-11ea-8ea7-9aa7a0569dbe.png)

Reply:
![Screen Shot 2020-03-31 at 1 45 28 PM](https://user-images.githubusercontent.com/4873481/78073522-e8995a00-7355-11ea-823b-a2d9dda67618.png)